ironfish/src/network/messages/newPooledTransactionHashes.ts

```diff
@@ -0,0 +1,51 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at https://mozilla.org/MPL/2.0/. */
+import bufio from 'bufio'
+import { TransactionHash } from '../../primitives/transaction'
+import { NetworkMessageType } from '../types'
+import { NetworkMessage } from './networkMessage'
+
+export class NewPooledTransactionHashes extends NetworkMessage {
+  hashes: TransactionHash[]
+
+  constructor(hashes: TransactionHash[]) {
+    super(NetworkMessageType.PooledTransactionsRequest)
+    this.hashes = hashes
+  }
+
+  serialize(): Buffer {
+    const bw = bufio.write(this.getSize())
+
+    bw.writeVarint(this.hashes.length)
+
+    for (const hash of this.hashes) {
+      bw.writeHash(hash)
+    }
+
+    return bw.render()
+  }
+
+  static deserialize(buffer: Buffer): NewPooledTransactionHashes {
+    const reader = bufio.read(buffer, true)
+    const length = reader.readVarint()
+    const hashes = []
+
+    for (let i = 0; i < length; i++) {
+      const hash = reader.readBytes(32)
```

Comment:
We usually use `readHash` from bufio for this -- I'm kind of open to changing this or creating our own readHash equivalent util, but it's convention in our codebase now

Reply:
fixed 👍🏻